docs/backend/upgrading/preparations.md

```diff
@@ -16,54 +16,45 @@ Things to do before you migrate Plone.
 ## Gather Information
 
 - Read the "What's new in..." for your relevant Plone version, and read the release notes.
-
-  - You'll find these in the CMFPlone directory of the distribution of the new version of Plone.
+  You'll find these on `dist.plone.org` in the release directory of the new version of Plone.
+  As an example, here are the [6.0.0b2 release notes](https://dist.plone.org/release/6.0.0b2/RELEASE-NOTES.md).
 
 - Make sure to check installed add-ons, if you **do not** need certain add-ons anymore, please deactivate and deinstall.
 
-- Check for dependencies
+- Check for dependencies.
 
   - Read the release notes for the Plone release you are upgrading to, in particular:
 
-  - What version of Python is required?
+    - What version of Python is required?
 
-  - What version of Zope is required?
+    - What version of Zope is required?
 
-  - Do you need any new Python libraries?
+    - Do you need any new Python libraries?
 
   - Make sure all the add-on products you are using have updated to support the version of Plone you are upgrading to.
 
   - Start with the third-party products that are in use on your site.
+    Verify that they have been updated or verified to work on the new version, and get them upgraded in your existing instance before you start the Plone/Zope/Python upgrade if possible.
 
-    - Verify that they have been updated or verified to work on the new version, and get them upgraded in your existing instance before you start the Plone/Zope/Python upgrade if possible.
-
-  - If Zope depends on a newer version of Python, install the new version of Python first.
-
-  - If the newer version of Plone depends on a newer version of Zope, you will need to install that before proceeding with the Plone upgrade.
```

Comment:
I agree that we should not duplicate documentation of Zope, and together with the updated note providing better references to migration, I like this change.